styles/theme.ts

```diff
@@ -0,0 +1,74 @@
+import { extendTheme } from '@chakra-ui/react'
+
+const global = {
+  styles: {
+    global: {
+      a: {
+        color: 'brand.100',
+      },
+    },
+  },
+}
+
+const fonts = {
+  body: 'Inter',
+  heading: 'Inter',
+}
+
+const colors = {
+  brand: {
```

Comment:
I think we may need to split this out into individual files soon-ish, but for speed keeping it all here.